plugins/scaffolder-actions/scaffolder-backend-module-utils/src/actions/jsonata/yaml.ts

```diff
@@ -17,14 +17,14 @@ import { createTemplateAction } from '@backstage/plugin-scaffolder-backend';
 import jsonata from 'jsonata';
 import { resolveSafeChildPath } from '@backstage/backend-common';
 import fs from 'fs-extra';
-import yaml from 'js-yaml';
-import { supportedDumpOptions, yamlOptionsSchema } from '../../types';
+import YAML from 'yaml';
+import { yamlOptionsSchema } from '../../types';
 
 export function createYamlJSONataTransformAction() {
   return createTemplateAction<{
     path: string;
     expression: string;
-    options?: supportedDumpOptions;
+    options?: any;
```

Comment:
Let's type this. This looks like it will introduce this modification as a breaking change unless we are able to match these old option types to the new ones

Reply:
I wasn't able to find this in the README, maybe it's fine as non-breaking if it's undocumented?

Reply:
I was also not able to find it either for `yaml` or anything remotely close to the provided options types in `js-yaml`

Reply:
applied the type back!